docroot/themes/humsci/humsci_basic/theme-settings.php

```diff
@@ -138,4 +138,16 @@ function humsci_basic_form_system_theme_settings_alter(array &$form, FormStateIn
     '#default_value' => theme_get_setting('global_footer_variant_classname'),
   ];
 
+  // Animation Enhancement
+  $form['options_settings']['humsci_basic_animation_enhancement'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Animation Enhancements'),
+  ];
+
+  $form['options_settings']['humsci_basic_animation_enhancement']['animation_enhancement'] = [
```

Comment:
instead of `$form['options_settings']['humsci_basic_animation_enhancement']['animation_enhancement']` maybe we could name that last value like so:

`$form['options_settings']['humsci_basic_animation_enhancement']['animation_toggle']`

Reply:
I like that! It makes more sense. :smile: